.github/actions/install-chrome/action.yml

```
@@ -0,0 +1,16 @@
+name: Install Chrome browser and driver for Testing
+description: Download and install the latest available Chrome for Testing and Chromedriver
+
+runs:
+  using: composite
+  steps:
+    - id: install-chrome
+      name: Install Chrome
+      shell: bash
+      run: |
+        sudo apt-get remove google-chrome-stable
+        npx @puppeteer/browsers install chrome@stable
+        npx @puppeteer/browsers install chromedriver@stable
+        sudo ln -s /home/runner/work/keycloak/keycloak/chrome/**/chrome-linux64/chrome /usr/bin/chrome
```

Since we're already in the same directory, perhaps we can replace this absolute path?
```suggestion
        sudo ln -s ./chrome/**/chrome-linux64/chrome /usr/bin/chrome
```